src/EditorFeatures/Core.Wpf/InlineRename/UI/Dashboard/RenameDashboard.xaml.cs

```diff
@@ -328,10 +329,15 @@ private void Apply_Click(object sender, RoutedEventArgs e)
             => Commit();
 
         private void Commit()
+        {
+            _ = CommitAsync();
+        }
+
+        private async Task CommitAsync()
         {
             try
             {
-                _model.Session.Commit();
+                await _model.Session.CommitAsync(previewChanges: false).ConfigureAwait(true);
```

Comment:
comment why CA(true).

Reply:
there is error reportin gto hte user below as well.  it's unclear to me which codepaths will show errors to the user, and if we'll do that multiple times.

Reply:
Will test & clean error reporting part.

Reply:
OK I doc why `ConfigureAwait(true);` and for the exception reporting, I just leave this logic unchanged for the legacy UI

Reply:
So I just leave this part unchanged. e.g. This is the legacy UI and we used to report the stack trace to user.